### PR TITLE
update getting started guide

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

**Comment by nshankar13** [comment]
@michelleN We don't want to remove the `> /dev/null 2>&1` because when we run that script in the pipeline, it would leak sensitive info about the subscription ID, and other info about the Azure Container Upstream CI subscription. The ADO project is currently public because we want all contributors to be able to see the CI builds, but we try to mask sensitive information from appearing in the pipeline log outputs in ADO.

File: docs/get-started.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ## Get Started
 Use the following scripts to publish chart, enable connected cluster, and add OSM Arc extension.
 
+## Prerequisites
+- Running Kubernetes cluster
```
Comment (ritazh) [inline docs/get-started.md:5]: Might be good to also mention this cluster should be created in a subscription and region that has the latest OSM arc extension available.

Comment (michelleN) [reply docs/get-started.md:5]: I was under the impression that the Kubernetes cluster can be running anywhere. It is the arc cluster that has to be in the subscription and region that has the latest OSM arc extension available.

Comment (ritazh) [reply docs/get-started.md:5]: > It is the arc cluster that has to be in the subscription and region

Technically, it's the region in which the connected cluster resource is created. Refer to https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/connect-cluster#create-a-resource-group

Comment (michelleN) [reply docs/get-started.md:5]: I misunderstood your comment. Thank you.

Comment (ritazh) [reply docs/get-started.md:5]: Not at all. Thanks for asking!

Comment (michelleN) [reply docs/get-started.md:5]: added a line about the region in the environment variables section. Let me know if that looks good to you.

Comment (ritazh) [reply docs/get-started.md:5]: Thanks for adding it! For private preview, only `eastus` and `westeurope` will be supported.

Comment (ritazh) [reply docs/get-started.md:5]: oops, sorry the link you referenced says exactly that.

**Review by ritazh — APPROVED** [review]

**Review by ksubrmnn — APPROVED** [review]

**Pull request merged**